src/components/data/data-utils.js

```diff
@@ -212,6 +213,9 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
       const workspace = Ajax().Workspaces.workspace(namespace, name)
       await (useFireCloudDataModel ? workspace.importEntitiesFile : workspace.importFlexibleEntitiesFile)(file)
       onSuccess()
+      Ajax().Metrics.captureEvent(Events.workspaceDataUpload, {
+        workspaceNamespace: namespace, workspaceName: name
+      })
```

Comment:
noting that we only log this event if the upload was successful. I assume that's your intention!

Reply:
@davidangb yes, that was my intention! If you disagree with that implementation I can change it, but I figured we wouldn't want to duplicate events if someone tries to upload a bunch of times with something not formatted correctly, etc.

Reply:
that works for me! If our business folks decide they want to track failures too, we can always change it later.